conans/model/options.py

```diff
@@ -286,15 +286,18 @@ def __init__(self, possible_values, name):
             self._possible_values = sorted(str(v) for v in possible_values)
 
     def __bool__(self):
-        if not self._value:
+        if not self.value:
```

Comment:
Why changing to use the property instead of the internal value directly? Seems like an extra indirection which is not necessary